keyboards/handwired/asesmero/2x6/rp2040/info.json

```diff
@@ -0,0 +1,20 @@
+{
+    "keyboard_name": "2x6 RP2040",
+    "processor": "RP2040",
+	"bootloader": "rp2040",
+	
+    "usb": {
+        "device_version": "0.0.2",
+    },
+	
+	"matrix_pins": {
+        "cols": ["GP2", "GP3", "GP4", "GP5", "GP6", "GP7"],
+        "rows": ["GP8", "GP9"]
+    },
+	
+	"rgblight": {	 
```

Comment:
```suggestion
    "ws2812": {
        "pin": "GP29"
    },
    "rgblight": {
```

Reply:
"ws2812": {
        "pin": "GP29"
    },

    "rgblight": {	 
         "led_count": 1,
    },
    
  This don´t compile :(

Reply:
The indents are correct now.

Reply:
Yeah, on master, it won't compile, but on develop, it will.  Though develop is getting merged into master in a week.

Having both should be okay,

Reply:
Ok.